integrations/AdobeAnalytics/browser.js

```diff
@@ -9,6 +9,7 @@ import * as heartbeatUtils from "./heartbeatHandle";
 import { getHashFromArray } from "../utils/commonUtils";
 import ScriptLoader from "../ScriptLoader";
 import logger from "../../utils/logUtil";
+import { commonNames } from "../integration_cname";
```

Comment:
This way we can detect errors at build time.
```suggestion
import { commonNames.ADOBE_ANALYTICS as NAME } from "../integration_cname";
```